src/gt4py/next/iterator/embedded.py

```diff
@@ -150,36 +150,6 @@ def _dace_descriptor(self) -> NoReturn:  # type: ignore[misc]
     __descriptor__ = _dace_descriptor
 
 
-@dataclasses.dataclass(frozen=True)
```

Comment:
Moved to `next.otf.arguments`, it seemed in better company there (more cohesion).